research.md

```diff
@@ -7,19 +7,7 @@
 
 <h2>Publications by Year</h2>
 
-<!-- Super important: do not indent the content line, because then it -->
-<!-- gets treated like code -->
-<!-- Old stuff now, Alex has a better suggestion
-{% assign sortedyrs = (site.publications | sort: 'name') | reverse %}
```

Comment:
My local jekyll was still trying to run these lines even though they are commented. This was causing a couple of minor warnings to popup.